### PR TITLE
chore: add realtime flag and env vars

### PR DESCRIPTION
**Pull request by elseagle**

### CODE REVIEW

**Comment by elseagle** [comment]
The AWS SECRET and KEY used here are placeholders till we get confirmation from @johnseekins 

cc: @bfossen-ce

File: environments.yml [hunk 1]
```diff
@@ -45,3 +45,9 @@ scrapers:
     paramstore: /bobsled/scrapers/NEW_YORK_API_KEY 
   - variable: DC_API_KEY
     paramstore: /bobsled/scrapers/DC_API_KEY
+  - variable: AWS_ACCESS_KEY_ID
+    paramstore: /bobsled/backups/AWS_ACCESS_KEY_ID
```
Comment (johnseekins) [inline environments.yml:49]: I think `/bobsled/scrapers/` is the more appropriate path here.

File: tasks/ma.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MA-scrape:
   image: openstates/scrapers
-  entrypoint: "poetry run os-update ma bills sort=latest page_limit=20"
+  entrypoint: "poetry run os-update ma bills sort=latest page_limit=2 --realtime"
```
Comment (bfossen-ce) [inline tasks/ma.yml:3]: Just curious why are you changing this from page_limit=20 to page_limit=2 ?

Comment (elseagle) [reply tasks/ma.yml:3]: OOps, no I'm not good catch

File: environments.yml (hunk 1 above)
Comment (elseagle) [reply environments.yml:49]: Roger that

**Review by bfossen-ce — APPROVED** [review]
LGTM

**Comment by johnseekins** [comment]
This _cannot_ be merged until we fully release the changes made in `openstates-core`. We'll need a pypi release and corresponding update to dependencies in `openstates-scrapers`.

**Comment by elseagle** [comment]
Roger that @johnseekins - makes sense.

**Pull request merged**